### PR TITLE
Update task page template

### LINKED ISSUE #140: Revisit/update task template
In ./pages/your_tasks/TEMPLATE_your_tasks.md.

"Training mentioned on the page" section (currently not existing) and "Tools and resources mentioned on the page" section (currently exists) can be added automaticaly and do not need to be listed manually in the template.

We may also decide to do something else with the template and add "Indicators mentioned on the page" section too.

And make sure that existing pages align with the new task template.

See duplicate issue #161.

### PR DESCRIPTION
**Pull request by anenadic**
Fixes #140.

### CODE REVIEW

File: pages/your_tasks/TEMPLATE_your_tasks.md [hunk 1]
```diff
@@ -1,56 +1,71 @@
...
+Briefly describe the use of specific tools or resources to solve the problem - e.g. using a bullet point list. 
+
+- Solution 1
+  - Sub-point
+- Solution 2
 
-* Bullet point solution 1
-  * Sub-point
-* Bullet point solution 2
+Make sure to add the tools and resources mentioned in the text in the "tools and resources" data file and then
```
Comment (egpbos) [inline pages/your_tasks/TEMPLATE_your_tasks.md:39]:
```suggestion
Make sure to add the tools and resources mentioned in the text in the `_data/tool_and_resource_list.yml` file and then
```

File: pages/your_tasks/TEMPLATE_your_tasks.md [hunk 2]
```diff
@@ -1,56 +1,71 @@
...
-## Tools and resources <!-- do not delete this heading and write your text below it -->
-List of relevant tools and resources for this task.
+## Concrete problem 3, formulated as a question <!-- example: how to use GitHub to share code ? -->
+
+### Description <!-- do not delete this heading and write your text below it -->
+
+### Considerations <!-- do not delete this heading and write your text below it -->
+
+### Solutions <!-- do not delete this heading and write your text below it -->
+
```
Comment (egpbos) [inline pages/your_tasks/TEMPLATE_your_tasks.md:60]:
```suggestion
```

The third example is a bit redundant, perhaps. I don't feel strongly about this, up to you.

**Review by egpbos — APPROVED** [review]
Looks good to me overall! Two small suggestions, up to you to include or not.

File: pages/your_tasks/TEMPLATE_your_tasks.md (hunk 2 above)
Comment (anenadic) [reply pages/your_tasks/TEMPLATE_your_tasks.md:60]: Yes, we did have 2 examples originally then I added another one - you are probably right, I'll shorten it, thx @egpbos.

**Comment by anenadic** [comment]
I am going to work on "Hot wo cite this page" as an addition to the ELIXIR Toolkit Theme (ETT) - so it gets included automatically on the page. Then we can remove the section from this template. I am merging this PR now but will create a separate issue for the ciration section and will link to the PR to ETT once I make it.

**Comment by anenadic** [comment]
> I am going to work on "How to cite this page" as an addition to the ELIXIR Toolkit Theme (ETT) - so it gets included automatically on the page. Then we can remove the section from this template. I am merging this PR now but will create a separate issue for the ciration section and will link to the PR to ETT once I make it.

An update - there is now a PR to the ETT to automatically add citation info: https://github.com/ELIXIR-Belgium/elixir-toolkit-theme/pull/315.

**Pull request merged**